modules/basegl/src/processors/axisalignedcutplane.cpp

```diff
@@ -29,150 +29,109 @@
-#include <inviwo/core/ports/datainport.h>                      // for DataInport
-#include <inviwo/core/ports/imageport.h>                       // for BaseImageInport, ImageOutport
-#include <inviwo/core/processors/processor.h>                  // for Processor
-#include <inviwo/core/processors/processorinfo.h>              // for ProcessorInfo
-#include <inviwo/core/processors/processorstate.h>             // for CodeState, CodeState::Expe...
-#include <inviwo/core/processors/processortags.h>              // for Tags, Tags::GL
-#include <inviwo/core/properties/boolproperty.h>               // for BoolProperty
-#include <inviwo/core/properties/cameraproperty.h>             // for CameraProperty
-#include <inviwo/core/properties/invalidationlevel.h>          // for InvalidationLevel, Invalid...
-#include <inviwo/core/properties/optionproperty.h>             // for OptionPropertyOption, Opti...
-#include <inviwo/core/properties/ordinalproperty.h>            // for FloatVec4Property, FloatPr...
-#include <inviwo/core/properties/propertysemantics.h>          // for PropertySemantics, Propert...
-#include <inviwo/core/properties/transferfunctionproperty.h>   // for TransferFunctionProperty
-#include <inviwo/core/rendering/meshdrawer.h>                  // for MeshDrawer
-#include <inviwo/core/rendering/meshdrawerfactory.h>           // for MeshDrawerFactory
-#include <inviwo/core/util/formats.h>                          // for DataFormatBase
-#include <inviwo/core/util/glmvec.h>                           // for vec3, vec4
-#include <modules/opengl/inviwoopengl.h>                       // for glTexParameteri, GL_TEXTUR...
-#include <modules/opengl/openglutils.h>                        // for GlBoolState, PolygonModeState
-#include <modules/opengl/shader/shader.h>                      // for Shader, Shader::Build
-#include <modules/opengl/shader/shaderutils.h>                 // for setUniforms, ImageInport
-#include <modules/opengl/texture/textureunit.h>                // for TextureUnitContainer
-#include <modules/opengl/texture/textureutils.h>               // for VolumeInport, activateAndC...
-#include <modules/opengl/volume/volumeutils.h>                 // for bindAndSetUniforms
-
-#include <algorithm>   // for min
-#include <functional>  // for __base
+#include <inviwo/core/algorithm/boundingbox.h>
+#include <inviwo/core/datastructures/buffer/buffer.h>
+#include <inviwo/core/algorithm/markdown.h>
+#include <inviwo/core/network/networklock.h>
+#include <modules/opengl/openglutils.h>
+#include <modules/opengl/shader/shaderutils.h>
+#include <modules/opengl/texture/textureunit.h>
+#include <modules/opengl/texture/textureutils.h>
+#include <modules/opengl/volume/volumeutils.h>
+#include <modules/opengl/rendering/meshdrawergl.h>
 
 namespace inviwo {
 
+namespace {
+
+const std::vector<OptionPropertyIntOption> channelsList = {{"channel1", "Channel 1", 0},
+                                                           {"channel2", "Channel 2", 0},
```

Comment:
Should not the values be 0-3, or 1-4?

Reply:
Probably yes :) Good catch! There were three more processors affected by the same problem.